iset.mm

```diff
@@ -136192,6 +136204,25 @@ S C_ ( P ( ball ` D ) T ) ) $=
           -> ( abs ` ( ( f ` z ) - y ) ) < e ) ) ) } ) $.
   $}
 
+  ${
+    $d s f x z w $.
+    $( Define the derivative operator on functions on the reals.  This acts on
```

Comment:
This definition is not only "on the reals".

Reply:
Fixed in set.mm and iset.mm.